tools/serve/serve.py

```diff
@@ -19,10 +17,11 @@
 import uuid
 from collections import defaultdict, OrderedDict
 from itertools import chain, product
+from typing import ClassVar, List, Set, Tuple
 
-from localpaths import repo_root
+from localpaths import repo_root  # type: ignore
```

Comment:
What's the error here? Seems like we should be able to mark this as a str.

Reply:
The error is "Cannot find implementation or library stub for module named 'localpaths'"

This kind of thing has come up in many places, and I'm thinking that perhaps together with https://github.com/web-platform-tests/rfcs/issues/82 we could think of a path to having no `sys.path` meddling and having all imports resolve correctly for mypy, but I'm not sure now what it would entail.

Does that sound alarming, or does your review stand?